templates/4b_normalize_image_data.ipynb

```diff
@@ -0,0 +1,142 @@
+{
```

Comment:
<p>I'd recommend splitting this cell up into two cells:</p><p> </p><ol><li>Making the directories to hold the normalized images and associated processing files.</li><li>The loop over each FOV to generate the corresponding MPH values.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>